### PR TITLE
guy/add widget tab

### PR DESCRIPTION
**Pull request by guyyoo**

### CODE REVIEW

File: src/rootPages/Designer/editors/views/ABViewTab.js [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+                  container: $component.getMultiview().$view,
+                  view: "template",
+                  autoheight: false,
+                  height: 1,
+                  width: 0,
+                  template:
+                     '<div class="ab-component-tools ab-layout-view ab-tab-tools">' +
+                     '<i class="fa fa-trash ab-component-remove"></i>' +
+                     '<i class="fa fa-edit ab-component-edit"></i>' +
+                     "</div>",
```
Comment (wongpratan) [inline src/rootPages/Designer/editors/views/ABViewTab.js:189]: Should we use ES6 string format ?

File: src/rootPages/Designer/editors/views/ABViewTab.js [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+                  container: $component.$view,
+                  view: "template",
+                  autoheight: false,
+                  height: 1,
+                  width: 0,
+                  template:
+                     '<div class="ab-component-tools ab-layout-view ab-tab-tools">' +
+                     '<i class="fa fa-trash ab-component-remove"></i>' +
+                     '<i class="fa fa-edit ab-component-edit"></i>' +
+                     "</div>",
```
Comment (wongpratan) [inline src/rootPages/Designer/editors/views/ABViewTab.js:210]: ^_^

**Review by wongpratan — APPROVED** [review]

File: src/rootPages/Designer/ab_work_object_list.js [hunk 3]
```diff
@@ -1,696 +0,0 @@
...
- * Manage the Object List
- *
- */
-
-const ABComponent = require("../classes/platform/ABComponent");
-const ABListNewObject = require("./ab_work_object_list_newObject");
-const ABListEditMenu = require("./ab_common_popupEditMenu"); // "./ab_work_object_list_popupEditMenu"
-
-module.exports = class AB_Work_Object_List extends ABComponent {
-   //.extend(idBase, function(App) {
```
Comment (nh758) [inline src/rootPages/Designer/ab_work_object_list.js:13]: Don't we still need these files?

File: src/rootPages/Designer/interface_common/ui_tab_form_popup.js [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+            position: "center",
+            head: " ",
+            body: {
+               id: ids.popupTabManagerForm,
+               view: "form",
+               elements: [
+                  {
+                     view: "text",
+                     name: "id",
+                     label: L("ID"),
```
Comment (nh758) [inline src/rootPages/Designer/interface_common/ui_tab_form_popup.js:46]: Is this tab Id? Do we need to show it here?

File: src/rootPages/Designer/properties/views/ABViewTab.js [hunk 5]
```diff
@@ -0,0 +1,227 @@
+/*
+ * ABViewCarousel
+ * A Property manager for our ABViewCarousel definitions
```
Comment (nh758) [inline src/rootPages/Designer/properties/views/ABViewTab.js:3]: change to ABViewTab?

**Review by nh758 — APPROVED** [review]
Looking good, a couple small things:

**Pull request merged**